elasticai/creator/nn/quantized_grads/base_modules/sigmoid.py

```diff
@@ -1,7 +1,7 @@
 import torch
 from torch.nn import Sigmoid as _Sigmoid
 
-from elasticai.creator.nn.quantized_grads._math_operations import MathOperations
+from .. import MathOperations
```

Comment:
Not referring to a leaf module here seems dangerous to me. The problem is: to load the parent module, python will typically have to load child modules, if someone decides to add add a symbol from `sigmoid` to the `base_modules` module, we get a circular import. Therefore we should keep symbols exported by `__init__.py` files for external clients and not import them in child modules.